lib/index.js

```diff
@@ -882,7 +882,7 @@ exports.isInteger = function (value) {
 
     return (typeof value === 'number' &&
             parseFloat(value) === parseInt(value, 10) &&
```

Comment:
This line can be omitted as well, as only integers will pass the `Number.isSafeInteger()` test.